twitchApi.js

```diff
@@ -0,0 +1,604 @@
+const {
+	//getNoAllowedChannelIdError,
+	//getChannelNotAllowedError,
+	//getBroadcaster,
+	//getBroadcasterId,
+	buildPollChoices,
+	toDiscordTimestamp
+} = require('./util.js');
+
+async function getUser(login) {
+	return (await fetch(`https://api.twitch.tv/helix/users?login=${login}`, {
+		headers: {
+			'Client-ID': process.env.TWITCH_CLIENT_ID,
+			'Authorization': `Bearer ${tokens.access_token}`
+		}
+	}).then(res => res.json()).catch(err => console.error)).data[0];
+}
+
+// https://dev.twitch.tv/docs/api/reference#get-polls
+async function getPoll(clientId, accessToken, broadcasterId) {
+	return new Promise(async (resolve, reject) => {
+		try {
+			const res = await fetch(`https://api.twitch.tv/helix/polls?broadcaster_id=${broadcasterId}`, {
+				method: 'GET',
+				headers: {
+					'Client-ID': clientId,
+					'Authorization': `Bearer ${accessToken}`,
+					'Content-Type': 'application/json'
+				}
+			});
+			const json = await res.json();
+			if (!res.ok) {
+				switch (res.status) {
+					case 400:
```

Comment:
Don't copy-paste error handling switch everywhere and extract it into a function instead.